iceoryx_examples/iceoptions/README.md

```diff
@@ -0,0 +1,68 @@
+# iceoptions
+
+## Introduction
+
+This example demonstrates what kind of quality of service options can be configured on the publisher and subscriber
+side. The options can be used for both, typed and untyped API flavours.
+
+## Expected output
+
+<!-- Add asciinema link here -->
```

Comment:
Either add the asciinema here or get rid of this section otherwise it looks like we have forgot to add the documentation here and that this file is either out of date or incomplete

Reply:
Will be done, once all examples are finalized.